portmidi/pm_linux/pmlinuxalsa.c

```diff
@@ -438,7 +438,7 @@ static PmError alsa_write(PmInternal *midi, PmEvent *buffer, long length)
 static PmError alsa_write_flush(PmInternal *midi, PmTimestamp timestamp)
 {
     alsa_descriptor_type desc = (alsa_descriptor_type) midi->descriptor;
-    VERBOSE printf("snd_seq_drain_output: 0x%x\n", (unsigned int) seq);
+    VERBOSE printf("snd_seq_drain_output: 0x%lx\n", (size_t) seq);

```

Comment:
I think this should be changed : whenever we printf `seq`, since seq is a pointer we should `%x` it

Reply:
And then we don't need to cast to something else

Reply:
Good idea!